elixir/Changelog.md

```diff
@@ -1,5 +1,35 @@
 # App Identity for Elixir Changelog
 
+## 1.2.0 / 2023-04-20
+
+- Add support for header groups in `AppIdentity.Plug` to better handle fallback
+  headers. Kinetic’s original Elixir implementation always verified only the
+  _first_ value from a _list_ of headers, like so:
+
+  ```elixir
+  with [] <- Conn.get_req_header(conn, "header-1"),
+       [] <- Conn.get_req_header(conn, "header-2"),
+       [] <- Conn.get_req_header(conn, "header-3") do
+    :error
+  else
+    [value | _] -> {:ok, value}
+  end
+  ```
+
+  AppIdentity.Plug always processes all values of a header and puts the result
+  in a map with the header name as the key, it meant that we would have had to
```

Comment:
I'm not sure what "…, it meant that we would have had to check these results in turn." means here, it looks like the remains of some editing.

I'd go for front loading the sentence again and say something like

> `header_groups ` allow you to group related headers under a single result key. 
> This means you do not have to check all of the related headers individually.

Reply:
Check the documentation on that one to see if the distinction between `headers` and `header_groups` makes sense. If it does, then I’ll probably leave this changelog entry unchanged since it’s explaining *why* `header_groups`, not *what* `header_groups`.

The *why* boils down to "if you have fallback headers like we do, not grouping them is a world of hurt because you have to check *all* the headers individually.